app/admin/settings/Amounts.scala

```diff
@@ -0,0 +1,22 @@
+package admin.settings
+
+import io.circe.{Decoder, Encoder}
+
+case class Amount(value: String, isDefault: Option[Boolean])
+case class AmountsRegions(
+  GBPCountries: List[Amount],
+  UnitedStates: List[Amount],
+  EURCountries: List[Amount],
+  AUDCountries: List[Amount],
+  International: List[Amount],
+  NZDCountries: List[Amount],
+  Canada: List[Amount]
+)
+case class Amounts(ONE_OFF: AmountsRegions, MONTHLY: AmountsRegions, ANNUAL: AmountsRegions)
+
+object Amounts {
+  import io.circe.generic.auto._
+
+  implicit val amountsEncoder = Encoder[Amounts]
+  implicit val amountsDecoder = Decoder[Amounts]
```

Comment:
oviously this is an existing project, however I normally use the convention to prepend the class name with "Wire" if it is used to generate a serialiser (e.g. `WireAmounts`, to clearly distinguish it from Domain abstraction level case classes and also make it more clear where renaming is safe.  In many cases we keep the domain model and wire model the same to reduce the amount of code.